artifacts/AdditionalSetup.ps1

```diff
@@ -58,6 +58,12 @@ if (Test-Path "$roleTailoredClientFolder") {
     Import-Module "$roleTailoredClientFolder\Microsoft.Dynamics.Nav.Ide.psm1" -Force -ErrorAction SilentlyContinue -DisableNameChecking
 }
 
+if (Test-Path 'c:\run\cosmo.compiler.helper.psm1')
```

Comment:
wouldn't it make sense to also only do this if it is a build container? It probably doesn't take long, but we definitely won't need it if it isn't a build container, right?

Reply:
yes please, to ensure that we really don't have any side effects